server/api/interfaces/error.go

```diff
@@ -1,6 +1,45 @@
 package interfaces
 
+type WsErrorCode uint64
+
 type WsError struct {
-	Code    uint64  `json:"code"`
-	Message string `json:"message"`
+	Code    WsErrorCode `json:"code"`
+	Message string      `json:"message"`
+}
+
+type HandlerError struct {
+	Code          WsErrorCode
+	InternalError error
+}
```

Comment:
i recommend implementing `Error() string` that way you'll implement `error` interface